src/pudl/metadata/resources/eia861.py

```diff
@@ -602,13 +609,14 @@
         "etl_group": "eia861",
     },
     "core_eia861__yearly_operational_data_revenue": {
-        "description": (
-            """A utility's revenue by type of electric operating revenue.
+        "description": {
+            "additional_summary_text": "utility revenue by type of electric operating revenue.",
+            "additional_details_text": """A utility's revenue by type of electric operating revenue.
```

Comment:
Duplicated sentence on purpose or accident? I could see either way
```suggestion
            "additional_details_text": """
```